shuup/front/admin_module/carts/views/_list.py

```diff
@@ -58,7 +59,8 @@ def get_queryset(self):
         """
         Ignore potentially active carts, displaying only those not updated for at least 2 hours.
         """
-        cutoff = now() - datetime.timedelta(hours=2)
+        hours = configuration.get(self.request.shop, "cart_update_delay", 0) or 0
```

Comment:
Instead `self.field_name` define variable `"CART_UPDATE_DELAY_CONF_KEY" = "shuup_front_cart_update_delay"`. Use that variable then here and `form_parts.py`. No need to `or 0` (that first zero is already default)